docs/en_US/quick_start_docker.md

```diff
@@ -5,7 +5,7 @@
 2. Set eKuiper source to an MQTT server. This sample uses server locating at ``tcp://broker.emqx.io:1883``. ``broker.emqx.io`` is a public MQTT test server hosted by [EMQ](https://www.emqx.io).
 
    ```shell
-   docker run -p 9081:9081 -d --name kuiper -e MQTT_SOURCE__DEFAULT__SERVERS=[tcp://broker.emqx.io:1883] lfedge/ekuiper:$tag
+   docker run -p 9081:9081 -d --name kuiper -e MQTT_SOURCE__DEFAULT__SERVER="tcp://broker.emqx.io:1883" lfedge/ekuiper:$tag
```

Comment:
Refer to https://docs.emqx.cn/broker/v4.3/rule/rule-engine.html#%E8%BF%81%E7%A7%BB%E6%8C%87%E5%8D%97 to add a section for migration. Because in docker, some users may still use the old version